bindings/python/setup.py

```diff
@@ -132,7 +134,7 @@ def get_version():
 def _build_mata():
     """Builds mata library"""
     with subprocess.Popen(
-        shlex.split("make release"),
+        shlex.split(f"make release-small BUILD_DIR={mata_build_dir}"),
```

Comment:
You mean for the target? I would not use `release-binding-python`, I think it can be useful generally, if we just want to build the library without anything else. I can change it to `release-lib`.

Reply:
Done, I also added `debug-lib` target.